printer_mods/Badnoob/AB-BN/Readme.md

```diff
@@ -114,61 +112,63 @@ I use the standard Voron print settings, but with 30% infill. I have gone as low
 
 Everyone will need:
 
-* AB-BN-30_Front
-* AB-BN-30_Back
-* AB-BN-28_Spring
-* AB-BN-28_Spacer
+* AB-BN-30_fan_front.stl
+* AB-BN-30_fan_back.stl
+* AB-BN-28_spring.stl
+* AB-BN-28_spacer.stl
 
   ##### Depending on your hotend you will need:
 
   #####   E3DV6:
-* AB-BN-28_Hotend-E3Dv6-Front
-* AB-BN-28_Hotend-E3Dv6-Back
+* AB-BN-28_Hotend-E3Dv6-front.stl
+* AB-BN-28_Hotend-E3Dv6-back.stl
 
   ##### Dragon (high flow and regular):
-* AB-BN-30_Hotend-Dragon-Front
-* AB-BN-28_Hotend-Drag-Back
+* AB-BN-30_Hotend-Dragon-front.stl
+* AB-BN-28_Hotend-Dragon-back.stl
 
   ##### Mosquito (high flow and regular):
 
-* AB-BN-28_Hotend-Mos-Front
-* AB-BN-28_Hotend-Mos-Back
-
-(special note:  I don't reccomend the Mosquito.  Its block design is problematic, but even more problematic are their business practices)
+* AB-BN-28_Hotend-Mos-front.stl
+* AB-BN-28_Hotend-Mos-back.stl
 
 # Assembly:
 
 ![This is the fan after modification](ModifiedFan.png)
 
-Step 1: Cut the ears off the 5015 fan. In order to fit, you must cut the ears (mounting holes) off the 5015 fan, and take off the top cover. the ears can be cut with diagonal clippers, hacksaw, bread knife, belt sander...whatever you have at hand. just make it look like the image above. If you cut too much its probably fine, as long as you don't damage the turbine.\
+Step 1: Cut the ears off the 5015 fan. In order to fit, you must cut the ears (mounting tabs) off the 5015 fan, and take off the top cover. The ears can be cut with diagonal clippers, hacksaw, bread knife, belt sander...whatever you have at hand. Just make it look like the image above. If you cut too much it's probably fine, as long as you don't damage the turbine.\
 \
-I have included stls for a de-earing tool. Its basically a handle/saw guide to put over the mounting tabs
+I have included stls for an optional de-earing tool (5015-deearing-tool and 5015-deearing-tool-b). The tool is meant as an handle and guide to saw off the ears. The tool fits around the ear and it's edge can be used as a saw guide to cut the mounting ear off.
```

Comment:
Replace `as an handle` with `as a handle`.

Replace `it's edge` with `its edge`.

Reply:
corrected, thank you.